.ci/integTestGen/src/get_target_branch.jl

```diff
@@ -20,10 +20,16 @@ function get_target_branch()::AbstractString
         error("Environment variable CI_COMMIT_REF_NAME is not set.")
     end
 
+    if ENV["CI_COMMIT_REF_NAME"] == "main"
+        # run on main branch
+        return "main"
+    end
```

Comment:
Does this trigger when running *on* the main branch or when running on a PR that is *targeting* the `main` branch?
At least the comment sounds like it's the first one, but it should be the second.

Reply:
When it runs on the branch main.

From the doc: `CI_COMMIT_REF_NAME` : `The branch or tag name for which project is built.`

https://docs.gitlab.com/ee/ci/variables/predefined_variables.html

Also GitLab, the mirror does not open an Merge Request. It simply creates a new branch, push the code on it and returns the result from testing the last commit.

Reply:
Okay I looked at the entire file now and I think I understand what's happening.